src/main/resources/jupyter/init-actions.sh

```diff
@@ -214,6 +215,11 @@ if [[ "${ROLE}" == 'Master' ]]; then
       docker cp /etc/${JUPYTER_GOOGLE_SIGN_IN_JS} ${JUPYTER_SERVER_NAME}:${JUPYTER_USER_HOME}/.jupyter/custom/
     fi
 
+    if [ ! -z ${JUPYTER_NOTEBOOK_CONFIG} ] ; then
+      log 'Installing Google sign in extension...'
+      gsutil cp ${JUPYTER_NOTEBOOK_CONFIG} /etc
+      docker cp /etc/${JUPYTER_GOOGLE_SIGN_IN_JS} ${JUPYTER_SERVER_NAME}:$/etc/jupyter/
```

Comment:
Typo, `JUPYTER_GOOGLE_SIGN_IN_JS` instead of `JUPYTER_NOTEBOOK_CONFIG`. Probably should use the pattern above with `gsutil cp`, `basename`, then `docker cp`.